### ISSUE TITLE
Oblique labels at small sizes less readable with new fonts

### ISSUE THREAD

**Issue by imagico** [gravitystorm/openstreetmap-carto#2380]
I think that with the changed fonts the oblique labels (like for most areas and rivers) in latin script are often less readable than before, especially at small text sizes - mostly due to the changed, more condensed proportions and due to the different design of the 'a' character which is prone to be confused with an 'o' and is also not consistent with the normal non-oblique and bold fonts.

**Comment by daganzdaanda** [comment]
Well, the "single-story" _a_ is the usual choice for cursive fonts. It's often a sign of a "true cursive" as compared to just a "slanted" or "oblique" font.

(Aside: I'd love to have a real backward cursive available for water features, as seen here:
http://www.lexikaliker.de/2010/08/roemisch-linkskursiv/
But that might be to germanocentric...) 

Noto seems to look smaller than the previous font. So we might think about increasing some text sizes to get an optically equal size.

**Comment by imagico** [comment]
> Well, the "single-story" a is the usual choice for cursive fonts. It's often a sign of a "true cursive" as compared to just a "slanted" or "oblique" font.

Yes, but in general italic fonts are not particularly well readable at small sizes and in low resolution rendering.

> Noto seems to look smaller than the previous font.

Not necessarily in total height but as said the italic variant is more condensed and also has a bit more spacing between characters i think - making them better separated at small sizes, in particular with curved labels but at the same time reducing the size of shapes.

**Comment by talllguy** [comment]
[Here's an example at z15](http://osm.org/go/ZcKB~8IZ--?m=). I can see how the lowercase 'a' might looks like an 'o', especially on the green background of that pitch. Why not increase all of the font sizes by a few points? They're all pretty tiny.

![image](https://cloud.githubusercontent.com/assets/1827881/18897663/293d918c-84fa-11e6-9053-ec9a96ce8db8.png)

**Comment by sommerluk** [comment]
As @daganzdaanda said:
> Well, the "single-story" a is the usual choice for cursive fonts. It's often a sign of a "true cursive" as compared to just a "slanted" or "oblique" font.
So I do not think this is a particular problem for legibility.

Anyway, at https://github.com/googlei18n/noto-fonts/issues/493 there is discussed the glyph form of `U+0061 LATIN SMALL LETTER A` also in contrast to similar glyphs like `U+0251`, `U+0252` and `U+AB64`. Summary: Maybe we get a double-storey italic glyph for `U+0061` as a stylistic alternate in the next release. _If_ this is accessible by a _binary_ OpenType feature, we could use it in openstreetmap-carto, and I would be in favour of doing it.

**Comment by sommerluk** [comment]
Our font rendering has evolved a lot during the past years.(Our own code, Noto design and the font rendering engine).

@imagico Do you still see this as an issue? From a quick test, I found also the cursive labels quite readable. On the other hand, we also have no real alternative available. Therefore, I would rather close this issue.

**Comment by imagico** [comment]
I don't think this has changed in substance since this issue was opened - Differentiation between 'a' and 'o' at small sizes is still poor.

To me this is more something to keep in mind if we think of adding new label types or contemplating changes to existing labeling and less an issue that requires acute fixing.  As such the observations made would probably be something for a knowledge collection useful for future label design decisions.

The decision if to close this should IMO be based on if we have consensus to reduce use of oblique labels because of this or not. I don't have a strong opinion on this in either direction.

In #2692 we discussed one possibility to do that.  But this is all tricky with the different scripts we need to take into account.
